### DISCUSSION TITLE
Using Program called schrodinger, receiving this error to make a writeable directory....

### DISCUSSION THREAD

**Discussion by Jedimike11** [ohmyzsh/ohmyzsh#9899] [Help]
```
michael@michael-XPS-15-9560:~$ $SCHRODINGER/licadmin INSTALL -k {license goes here}
ERROR: Could not find a writable directory from the following standard directories to install license file - ('/opt/schrodinger/licenses', '/opt/schrodinger2020-4/licenses')
michael@michael-XPS-15-9560:~$ sudo mkdir -m {permissions} licenses
[sudo] password for michael: 
mkdir: invalid mode ‘{permissions}’
michael@michael-XPS-15-9560:~$ mkdir -m licenses
mkdir: missing operand
Try 'mkdir --help' for more information.
michael@michael-XPS-15-9560:~$ mkdir -m 
michael@michael-XPS-15-9560:~$ mkdir -m /opt/
google/            schrodinger/       schrodinger2020-4/ zoom/
michael@michael-XPS-15-9560:~$ mkdir -m /opt/schrodinger/licenses/
mkdir: missing operand
michael@michael-XPS-15-9560:~$ mkdir -m licenses
mkdir: missing operand
Try 'mkdir --help' for more information.
```

I receive this error when trying to install a license:  "ERROR: Could not find a writable directory from the following standard directories to install license file - ('/opt/schrodinger/licenses', '/opt/schrodinger2020-4/licenses')"

Here is the website of where I can get installation instructions. However it does not provide how to get a folder for reading and writing permissions, and I am unsure how to go about this. https://www.schrodinger.com/license-installation-instructions

In addition, I did create a folder in my /opt/schrodinger directory named licenses, just by doing:

```
cd /opt/schrodinger
mkdir license 
```

**Comment by Jedimike11** [comment]
Ignore the cross outs. idk why it did that

**Reply by rwmitchell** [reply]
All the places you see ```{something}``` you are supposed to replace ```{something}``` with the actual thing.

Your '$' prompt is typical of bash, not zsh.  Even if you are using zsh, this is an installation problem for a specific piece of software not likely to be used by most people.

The command you need to learn is ```man``` as in ```man mkdir```, then what you are doing will start to make more sense instead of just blindly trying random things.
